src/meta/src/hummock/manager/mod.rs

```diff
@@ -1388,39 +1388,47 @@ where
                         == sst.get_table_ids().len();
                 if is_trivial_adjust {
                     *compaction_group_id = *group_table_ids.first_key_value().unwrap().0;
-                    is_sst_belong_to_group_declared = true;
+                    // is_sst_belong_to_group_declared = true;
```

Comment:
nit:

Reply:
> nit:

This is deliberate in case someone may use this `is_sst_belong_to_group_declared` in the future.